applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -851,28 +432,19 @@ static void format_final_result(char *buf, size_t buf_len, size_t buf_max_len)
 	}
```

Comment:
Should line 430 be error, not warning? (LOG_WRN("No room to insert CRLF");)

Reply:
Unsure, I did not change this functionality, so suggesting keeping it as is.